programs/lib/linker/flexpret_app.ld

```diff
@@ -1,67 +1,75 @@
 OUTPUT_ARCH("riscv")
 
-INCLUDE flexpret_config.ld
+/*INCLUDE flexpret_config.ld*/
+INCLUDE flexpret_hwconfig.ld
+INCLUDE flexpret_swconfig.ld
+INCLUDE flexpret_bootloader_config.ld
+
+ISPM_BOOTLOADER_END = BOOTLOADER_APP_START_ADDR;
+
+/* TODO: Fix hardcoded 0x1000, potentially with some nm of the bootloader */
```

Comment:
Actually, I found that this was a non-issue on the cmake-port branch. I initially thought we would need to leave the bootloader's data as to not break the bootloader. But we can safely overwrite it with out application because the bootloader's data is copied into DSPM every time it initializes. This happens in the `Reset_Handler` function found in `startup.c`. So I'll just remove the comment and the hard-coded value instead.